service/src/main/java/uk/gov/hmcts/reform/rd/client/OrganisationApi.java

```diff
@@ -39,4 +39,10 @@ Organisation findUserOrganisation(
         @RequestHeader(AUTHORIZATION) String authorisation,
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorization
     );
+
+    @GetMapping("/refdata/internal/v1/organisations")
+    Organisation findManagedUserOrganisation(
```

Comment:
as discussed, this method should accept organisationId. I would rename it to match method above